PoGo.NecroBot.CLI/PoGo.NecroBot.CLI.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Exe</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>PoGo.NecroBot.CLI</RootNamespace>
-    <AssemblyName>PoGo.NecroBot.CLI</AssemblyName>
+    <AssemblyName>PoGo NecroBot</AssemblyName>
```

Comment:
Could you make it "NecroBot" or "PoGo.NecroBot"? I'm just imagining a bunch of script kiddies raising spam issues because they don't know how to escape whitespace.